app/controllers/api/comments_controller.rb

```diff
@@ -20,37 +21,35 @@ def create
     begin
       saved = comment.save!
     rescue ActiveRecord::StatementInvalid => invalid
-      return render_json_message(:forbidden, errors: "Invalid comment")
+      return render json: {message: 'Invalid comment'}
     end
     if saved
-      render_json_message(:ok, message: 'Comment successfully created!')
+      return render json: {message: 'Comment successfully created!'}
     else
-      render_json_message(:forbidden, errors: comment.errors.full_messages)
+      return render json: {error: comment.errors.full_messages}
     end
   end
 
   def destroy
     comment = Comment.find(params[:id])
     if comment.destroy
-      render_json_message(:ok, message: 'Comment successfully deleted!')
+      return render json: {message: 'Comment successfully deleted!'}
     else
-      render_json_message(:forbidden, errors: comment.errors.full_messages)
+      return render json: {error: comment.errors.full_messages}
```

Comment:
A couple things: you don't need the `return` in the controller cuz rails has an implicit return
also, do something like render :json => { :errors => comment.errors.full_messages }, :status => 422
because the current code will still return a 200 success